test_driver/app_test.dart

```diff
@@ -72,97 +80,169 @@ void main() {
      * Run Tests
      */
     test(
-      'open drawer menu',
+      '[open drawer menu and outdoor interest page]',
       () async {
         // Open drawer menu
         final SerializableFinder drawerMenu =
             find.byTooltip('Open navigation menu');
         await tap(driver, drawerMenu);
 
-        // Close drawer menu
-        await driver.scroll(
-            drawerMenu, -300.0, 0.0, const Duration(milliseconds: 500));
+        // Open outdoor interest page
+        await tap(driver, outdoorInterest);
+
+        // SGW tab
+        await tap(driver, sgwTab);
+        await driver.waitFor(card);
+        await delay(1000);
```

Comment:
Is there a way to wait until content is loaded? Fixed delays could introduce inconsistencies.

Reply:
The delays are only so slow it down for demo purposes. They normally would not be there.
As for the waiting of content load, methods like .waitFor and .pumpAndSettle ensure widgets are loaded before any action is taken.

Reply:
Perfect, makes sense